lib/adal-angular.js

```diff
@@ -320,7 +320,7 @@ if (typeof module !== 'undefined' && module.exports) {
                             if (authService.loginInProgress()) {
                                 authService.info('login already start.');
                                 return $q.reject();
-                            } else if (authService.config && isEndpoint) {
+                            } else if (authService.config && (!authService.config.endpoints || isEndpoint)) {
```

Comment:
@sarmut I've already talked with @tushargupta51, and we're in agreement with this change.